frontend/public/components/masthead-toolbar.jsx

```diff
@@ -225,6 +251,7 @@ class MastheadToolbar_ extends React.Component {
 
     return (
       <Dropdown
+        key="users"
```

Comment:
I'm unclear why we need `key` here.

Reply:
Removed it here, only needed it in `<DropdownItem ../>`'s